### ISSUE TITLE
Status Message Noise

### ISSUE THREAD

**Issue by enj** [aeirola/slack-irc-client#13]
The `Status messages delivered to Slackbot DM chat` feature is hard to deal with because there is no way to silence the Slackbot.  Thus channels that have constant status messages (i.e. git commits) cause a lot of notifications.  It would be nice if these could be sent elsewhere (I am confused as to why they don't show up in the actual channel they are associated with).

**Comment by aeirola** [comment]
Not really sure what kind of status messages you are referring to here. Currently the following messages are sent to the Slackbot DM:

 * MOTD messages
 * Notice messages
 * CTCP messages
 * Invites

These events are emitted by the node-irc library, as per the documentation at https://node-irc.readthedocs.io/en/latest/API.html#events

Would you be able to be a bit more specific about which kind of messages you would want to be routed differently?

**Comment by enj** [comment]
I believe it is the notice and CTCP messages that were generating the most traffic.  Some example messages:

```
Notice from github-rosie-bot
[13origin] 15juanvallejo opened pull request #9647: UPSTREAM: 0000: base64 decode struct values after parsing resource ob… (06master...06jvallejo_fix-secret-token-value-oc-get) 02https://git.io/vopeX 
```
```
CTCP notice from syncretism_work
ACTION high-fives stenwt 
```
```
Notice from [kubot]
[kubernetes] ncdc opened pull request #28294: Return immediately for singular get on error (master...return-error-for-kubectl-get-singluar) https://git.io/vopuV 
```

These would be useful if they showed up with the actual channel they are associated with (I believe this is the `to` parameter of the function) and didn't cause Slack to notify me unless I specifically requested it.

**Comment by aeirola** [comment]
Yeah, definitely makes sense. I'll try to look into it at some point. Thanks for pointing this out.

**Comment by aeirola** [comment]
Fixed by #16
